src/main/java/com/edu/Compare.java

```diff
@@ -0,0 +1,47 @@
+package com.edu;
+
+public class Compare {
+
+    public void compare(String myNum, String comNum) {
+
+        int strike = 0;
+        int ball = 0;
+
+        for (int i = 0; i < myNum.length(); i++) {
```

Comment:
위의 코드는 아래와 같이 myNum.length()를 변수로 추출해서 사용하는 것이 좋습니다. 
```suggestion
        int myNumLength = myNum.length();
        for (int i = 0; i < myNumLength; i++) {
```
그 이유가 무엇일지에 대해 고민해보면 좋을 것 같습니다. 
이전 코드에 비해서 어떤 점이 좋아졌을까요? 이전 코드는 어떤 문제점이 있을까요?